depends/README.md

```diff
@@ -41,7 +41,13 @@ The paths are automatically configured and no other options are needed.
 
 #### Common
 
-    apt install bison cmake curl make patch pkg-config python3 xz-utils
+    apt install cmake curl make patch
+
+#### GUI
+
+Skip the following packages if you don't intend to use the GUI and will build with [`NO_QT=1`](#dependency-options):
+
+    apt install bison g++ pkg-config python3 xz-utils
```

Comment:
If you're adding `g++` here, you can remove it from the macOS section below.

Reply:
Thanks! Removed.

Reply:
The user can use a different compiler to build depends, but Qt 5 still insists on GCC for building the native `qmake`.

Reply:
Okay, sure, then it makes sense.